### ISSUE TITLE
since upgrade to High Sierre phoneGap desktop completely blank

### ISSUE THREAD

**Issue by RaspChris** [phonegap/phonegap-app-desktop#872]
## _System Configuration_
![capture d ecran 2018-02-01 a 10 46 17](https://user-images.githubusercontent.com/24367325/35671994-4f47a126-073d-11e8-8370-a4af94d45db0.png)

### PhoneGap Desktop version
Latest

### Computer OS version
10.13.3

Nothing whatsoever works... Was working fine before upfdate from el Capitan.
Very sorry for this bad news.
Chris

**Comment by surajpindoria** [comment]
Hi @RaspChris,

Firstly could you try reinstalling the application. I am currently on High Sierra and it does work, so maybe something with the upgrade screwed things up. Also, please answer the following questions so we can help to debug this.

1. PhoneGap Desktop version / PhoneGap CLI version
1. PhoneGap Mobile App version
1. Mobile devices & Mobile OS version
1. Computer OS version
1. network type (home/corporate)
1. is your network using a proxy?
1. computer / network security (firewall / anti-virus / disk encryption)
1. are you using VMs on your computer?
1. do you have multiple network adapters (or virtual network adapters)
1. are you using VPN?

**Comment by RaspChris** [comment]
Thank you.

Yes  This is the first thing I did. I had reinstalled the desktop application after the update of my computer.
1)This is the latest version of phone Gap GUI that I have. (I do not have access to the info in about PhoneGap as nothing whatsoever works).
2 et 3) As to the mobile version : on android (Wiko, 5.1), it tries to download again and again., on iOS (IPhone, 11.2.5), shows the right IP server address, tries to connect and fails.
4) Computer is 10.13.3.
5)My network is home but I have fixed IP. This is not a network problem.
No proxy, Next questions answers are no.

When the application launches, the normal screen appears for a brief second (with the turning wheel), then it goes white and unresponsive.
Maybe I have to look into the CLI thing, but..... Well I'll try.
